automation/taskcluster/lib/tasks.py

```diff
@@ -106,6 +107,7 @@ def craft_assemble_raptor_task(self, variant):
             'echo "https://fake@sentry.prod.mozaws.net/368" > .sentry_token',
             'echo "--" > .adjust_token',
             'echo "-:-" > .leanplum_token',
+            'echo "" > .digital_asset_links_token',
```

Comment:
If you want to create an empty file, a more elegant way of doing it is `touch .digital_asset_links_token` :)

Reply:
I do that all the time in a CLI but here, I seem to have switched my brain off.. 🤦‍♂